examples/standalone/main.tf

```diff
@@ -0,0 +1,203 @@
+  provider = google-beta
+
+  project    = module.template_project.project_id
+  location   = local.location
+  repository = "python-modules"
+  role       = "roles/artifactregistry.reader"
+  member     = "serviceAccount:${module.secured_data_warehouse.dataflow_controller_service_account_email}"
+}
+
+resource "google_artifact_registry_repository_iam_member" "confidential_python_reader" {
+  provider = google-beta
+
+  project    = module.template_project.project_id
+  location   = local.location
+  repository = "python-modules"
+  role       = "roles/artifactregistry.reader"
+  member     = "serviceAccount:${module.secured_data_warehouse.confidential_dataflow_controller_service_account_email}"
+}
+
+// The sample data we are using is a Public Bigquery Dataset Table
+// that contains a United States Internal Revenue Service form
+// that provides the public with financial information about a nonprofit organization
+// (https://console.cloud.google.com/bigquery?project=bigquery-public-data&d=irs_990&p=bigquery-public-data&page=table&ws=!1m9!1m3!3m2!1sbigquery-public-data!2sirs_990!1m4!4m3!1sbigquery-public-data!2sirs_990!3sirs_990_ein&t=irs_990_ein)
+module "regional_deid_pipeline" {
+  source = "../../modules/dataflow-flex-job"
+
+  project_id              = module.base_projects.data_ingestion_project_id
+  name                    = "dataflow-flex-regional-dlp-deid-job-python-query"
+  container_spec_gcs_path = module.template_project.python_re_identify_template_gs_path
+  job_language            = "PYTHON"
+  region                  = local.location
+  service_account_email   = module.secured_data_warehouse.dataflow_controller_service_account_email
+  subnetwork_self_link    = module.base_projects.data_ingestion_subnets_self_link
+  kms_key_name            = module.secured_data_warehouse.cmek_data_ingestion_crypto_key
+  temp_location           = "gs://${module.secured_data_warehouse.data_ingestion_dataflow_bucket_name}/tmp/"
+  staging_location        = "gs://${module.secured_data_warehouse.data_ingestion_dataflow_bucket_name}/staging/"
+
+  parameters = {
+    query                          = "SELECT ein, name, ico, street, city, state, income_amt, revenue_amt FROM [bigquery-public-data:irs_990.irs_990_ein] LIMIT 10000"
+    deidentification_template_name = module.de_identification_template.template_full_path
+    window_interval_sec            = 30
+    batch_size                     = 1000
+    dlp_location                   = local.location
+    dlp_project                    = module.base_projects.data_governance_project_id
+    bq_schema                      = local.bq_schema_irs_990_ein
+    output_table                   = "${module.base_projects.non_confidential_data_project_id}:${local.non_confidential_dataset_id}.${local.non_confidential_table_id}"
+    dlp_transform                  = "DE-IDENTIFY"
+  }
+}
+
+resource "time_sleep" "wait_de_identify_job_execution" {
+  create_duration = "600s"
```

Comment:
can this be shortened with the python pipeline fix?

Reply:
The `create_duration = "600s"` already considers the pipeline fix.

Currently most of the time is dedicated to the machine turn on.